src/components/Badge/Badge.styles.ts

```diff
@@ -0,0 +1,34 @@
+import color from '../../styles/colors';
+import fonts from '../../styles/fonts';
+import resetCSS from '../../styles/reset';
+import styled from 'styled-components';
+import type { BadgeProps, colorState } from './types';
+
+type TStyleProps = Pick<BadgeProps, 'state'>;
+
+const getBackgroundColor = (type: colorState) => {
+    switch (type) {
+        case 'danger':
+            return color.red;
+        case 'success':
+            return color.green;
+        case 'warning':
+            return color.yellow;
+        default:
+            return color.blue;
+    }
+};
+
+export const DivStyled = styled.div<TStyleProps>`
+    ${resetCSS}
+    ${fonts.text}
+    width: min-content;
```

Comment:
`width` to `justify-content` can you sort alphabetically please.

We try to keep all props and CSS attribute in alphabetical sorted. But you nailed it with the imports first and the StyledComponent function last, bravo!